_posts/2019-10-01-021-programmable-money.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: post
+number: 21
+title: 'Programmable Money'
+date: 2019-10-01 07:00:00 +0200
+intro: "To figure out why blockchain programming is just programming, we talk with Gerald Bauer about his views on no-coiners and Bitcoin Maximalists, differences between Bitcoin and Ethereum, and finally how to solve the halting problem."
```

Comment:
Either the "No-Coiners" are capitalized (pun intended), or the "Maximalists" minimized (oh, wow, the jokes just write themselves here...) ;)

Reply:
done